prepare/metrics/llm_as_judge/binary_judge.py

```diff
@@ -52,6 +51,13 @@ def get_prediction_field(metric_type):
                     infer_log_probs=use_logprobs,
                 )
 
+                add_to_catalog(
+                    metric,
+                    f"metrics.rag.{metric_type}.{inf_label}_{template_short_name}{logprobs_label}",
+                    overwrite=True,
+                )
+
+                # for backwards compatibility: keep also legacy path to metrics
```

Comment:
This is an example, where we need references and deprecation options:
https://github.com/IBM/unitxt/issues/1341